### ISSUE TITLE
configure.sh does not support tensorflow installation using conda

### ISSUE THREAD

**Issue by MartinHahner88** [waymo-research/waymo-open-dataset#10]
https://github.com/waymo-research/waymo-open-dataset/blob/9c663111bb63a9520d7037bd1cb878b0210f7a1d/configure.sh#L44

**Comment by peisun1115** [comment]
It is not supported for now.  i am not familiar with conda. I can take a look in the future. For now, you can just change it in your client. What the configure.sh is doing is just to find where tensorflow is installed.

**Comment by nan0755** [comment]
You can just comment this line and use this code like this in your conda envs, it just ignore the 'if' condition:

```sh
# if [[ $(pip show tensorflow) == *tensorflow* ]] || [[ $(pip3 show tf-nightly) == *tf-nightly* ]] ; then
#   echo 'Using installed tensorflow'
  TF_CFLAGS=( $(python3 -c 'import tensorflow as tf; print(" ".join(tf.sysconfig.get_compile_flags()))') )
  TF_LFLAGS="$(python3 -c 'import tensorflow as tf; print(" ".join(tf.sysconfig.get_link_flags()))')"
# else
#  echo 'CPU Tensorflow is not installed'
# fi
```

**Comment by peisun1115** [comment]
I think zeyu@'s solution  should work. I am closing this issue. Feel free to re-open if you still see a problem. 

We have also released 1.0.1. You can now install directly from pip/pip3
